lo-la-do-li/devWit
# Feature/set joke card functions

### What’s this PR do?
As a user, I can now... 
- Click on a joke card in the Get Material view, and add it to My Set
- Remove a joke card from My Set (removes both from global state and the UI)
As a dev, I am able to... 
- Fetch data from an API to populate joke cards
- Store `myJokes` in global state, update on button clicks
### Where should the reviewer start?
- package.json: 13
- GetMaterial: 10
- JokeCard: 10
- JokesContainer: 9
- MySet: 6
- src/apiCalls, /store, /actions: line 1
### How should this be manually tested?
- `npm start`, navigate to GetMaterial, look for & add some cards 
- Navigate to MySet, see added cards
- Click the trash icon to remove cards from set
### Any background context you want to provide?
- GetMaterial and MySet are sibling components that each render a JokesContainer, passing it different props (addJoke & removeJoke, respectively) as suited to context 
- `npm -i use-global-hook` was installed in order to maintain state for MyJokes in MySet component 
### Screenshots (if appropriate):
n/a
### Questions:
n/a
